includes/multi-currency/Analytics.php

```diff
@@ -109,11 +109,17 @@ public function update_order_stats_data( array $args, $order ): array {
 			return $args;
 		}
 
-		$order_exchange_rate = floatval( $order->get_meta( '_wcpay_multi_currency_order_exchange_rate', true ) );
+		$stripe_exchange_rate = $order->get_meta( '_wcpay_multi_currency_stripe_exchange_rate', true )
+			? floatval( $order->get_meta( '_wcpay_multi_currency_stripe_exchange_rate', true ) )
+			: null;
+		$order_exchange_rate  = ( 1 / floatval( $order->get_meta( '_wcpay_multi_currency_order_exchange_rate', true ) ) );
 
-		$args['net_total']      = $this->convert_amount( floatval( $args['net_total'] ), $order_exchange_rate, wc_get_price_decimals() );
-		$args['shipping_total'] = $this->convert_amount( floatval( $args['shipping_total'] ), $order_exchange_rate, wc_get_price_decimals() );
-		$args['tax_total']      = $this->convert_amount( floatval( $args['tax_total'] ), $order_exchange_rate, wc_get_price_decimals() );
+		$exchange_rate = ! is_null( $stripe_exchange_rate ) ? $stripe_exchange_rate : $order_exchange_rate;
```

Comment:
Tiny nitpick here. As one-liner lover, I think this can be written like that:
```suggestion
		$exchange_rate =  $stripe_exchange_rate ?? $order_exchange_rate;
```

Sad we can't use the short ternary operator `?:` to rewrite the 3 lines `$stripe_exchange_rate` into just one 😄

Reply:
That works well, thanks for the suggestion! Will update before merging.